goodtables/checks/contrib/sequential_column.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+from __future__ import division
+from __future__ import print_function
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+from ...registry import check
+
+
+# Module API
+
+@check('sequential-column', type='custom', context='body')
+class SequentialColumn(object):
+
+    # Public
+
+    def __init__(self, column, **options):
+        self.__column = column
+        self.__cursor = None
+
+    def check_row(self, errors, columns, row_number):
+
+        # Get column
+        column = None
+        for item in columns:
+            if self.__column in [item['number'], item['header']]:
+                column = item
+                break
+
+        # Check column
+        if not column:
+            return errors.append({
+                'code': 'sequential-column',
+                'message': 'Sequential column violation',
```

Comment:
I've used it as a `contrib.checks` POC so I think it should be renamed to `sequential-integers` and all messages here will be rewritten to be more helpful.